checker/manual/troubleshooting.tex

```diff
@@ -741,9 +736,32 @@ \subsection{Build the Checker Framework\label{building}}
 \end{Verbatim}
 % $ to unconfuse Emacs LaTeX mode
 
+Once \<checker.jar> is built, you may wish to put the Checker Framework \<javac> even earlier on
+your path. The Checker Framework's \code{javac} ensures that \<checker.jar> 
+is on your classpath and bootclasspath, but is otherwise
+identical to the Type Annotations compiler:
+
+\begin{Verbatim}
+export PATH=$JSR308/checker-framework/checker/bin:$JSR308/jsr308-langtools/dist/bin:${PATH}
+\end{Verbatim}
+
+If you are developing Checkers and wish to use the latest builds of your source files compiled by
+Eclipse, you can use the development version of javac instead. This version will use compiled
```

Comment:
Why is this comment specific to Eclipse?  Doesn't it apply equally regardless of one's IDE?
If it is specific to Eclipse, please first state how to do this when building using the standard Checker Framework buildfile.